demos/python_demos/clip_image_classification/servable/preprocess.py

```diff
@@ -27,10 +28,24 @@ def initialize(self, kwargs: dict):
         self.processor = CLIPProcessor.from_pretrained(model_id)
 
     def execute(self, inputs: list):
-        image = Image.open(BytesIO(inputs[0]))
-        
-        input_labels = np.frombuffer(inputs[1].data, dtype=inputs[1].datatype)
-        text_descriptions = [f"This is a photo of a {label}" for label in input_labels]
+        if inputs[0].datatype != 'BYTES':
+            print("ERROR: Wrong datatype set for input tensor 1. Expected BYTES got: " + str(inputs[0].datatype))
+
+        if inputs[1].datatype != 'BYTES':
+            print("ERROR: Wrong datatype set for input tensor 2. Expected BYTES got: " + str(inputs[1].datatype))
+
+        if len(bytes(inputs[0])) != 206580:
```

Comment:
During Friday's review we agreed we do not want to overwhelm user with validation